css/style.css

```diff
@@ -668,3 +668,7 @@ address {
 .fl-theme-lgdg .floe .floe-image-cloud           { background: url("../images/lgdg-floe-05.png") no-repeat 0 0 / 325px 325px; }
 .fl-theme-lgdg .floe .floe-image-preferenceIcons { background: url("../images/lgdg-floe-04.png") no-repeat 0 0 / 325px 325px; }
 .fl-theme-lgdg .floe .floe-image-videoPlayer     { background: url("../images/lgdg-floe-03.png") no-repeat 0 0 / 325px 325px; }
+
+.highlighted-links{
+    font-size:.83em;
+}
```

Comment:
Did you add all these new classes just for styling purposes? 
Requisite classes have already been defined in the HTML code. This could have been done with CSS selectors only.

Reply:
Do you think making changes to font-size is the appropriate solution to this issue? abrupt changes in font-size is not an appropriate solution to prevent overlapping issues. You could have just changed the display property as per need.

Reply:
I think it is the best and most appropriate solution for this.

Reply:
Check #122